<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFile</class>
 <widget class="QMainWindow" name="SendFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SendFile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cmb_com"/>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_barate">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>921600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2250000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lb_pkrate">
        <property name="text">
         <string>包间隔(ms)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_pksize">
        <property name="text">
         <string>包大小(Byte)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_pkrate"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_pksize"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_filename"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fileload">
        <property name="text">
         <string>文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_filesend">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="pb_send">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_hexshow"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>324</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
